### DISCUSSION TITLE
Intra channel search

### DISCUSSION THREAD

**Discussion by jasontalbott** [rrirower/DP2SceneGraphWizard#102] [General]
Wondering if anyone is having success with the intra channel search? It looks like a newer feature for the wizard? For me it is returning results which is great, but when I click on the results to play them, I'm getting "error: got invalid pathname, load the next one. An unexpected problem (but not server timeout or HTTP error) has been detected." 

Even if I can't get the search to work correctly, I can't complain (too much, lol). This wizard has allowed me to make a much better channel than using the rdp to scenegraph template that ROKU came up with. Nice work and many thanks to rrirower for what you've done for us.

**Comment by rrirower** [comment]
What version are you running?

**Comment by jasontalbott** [comment]
It says 1.0.0.0 beta 3

**Reply by jasontalbott** [reply]
ooooohh.... you have a beta 5.... I will give that a try.

**Reply by rrirower** [reply]
You should add yourself as a "watcher" or "star" to this repo if you want to be notified when new versions are released.

**Comment by jasontalbott** [comment]
Ok, I used beta 5. I have mixed results. 

My feed has a movie at the beginning which I'm using to feature one video that the user can play with minimal clicks (just 2 clicks). The rest of my content is four series that have seasons and episodes. If the featured video (movie content type) is returned in the search results, I can play it from the search screen. If any of the episodes are returned I get the error as stated in my original post. 

I also notice that the "movie" shows up in the results with the correct description text, while the episodes show the series description instead of the episode description. 

I validated my feed using your Channel Feed Manager tool and it comes out green, so that could possibly be a bug? TLDR; search is working for movies, partially working for series/season/episodes (returns the content, but can't click through on it).

**Reply by rrirower** [reply]
Can you post the URL for your feed so I can test further?

**Reply by jasontalbott** [reply]
https://tw-roku-images.s3.amazonaws.com/Test+TW.json

**Reply by jasontalbott** [reply]
If I search the series name "US Telecasts", it shows in the results "Series: 0 Seasons with 0 Episodes"... when there's actually 12 seasons and 24 episodes (in the test feed anyway).

**Comment by jasontalbott** [comment]
I'm not sure if this is helpful but I spent this morning figuring out how to see the debug console (set up OpenSSH on my computer and enabled telnet) and when I click on one of the search results that is not the movie object, I got this in the debbuger:

```
=================================================================
Warning occurred while setting a field of an RoSGNode
-- Type mismatch occurred when setting the "focusedChild" field of a "ContentNode" node
   at line 171 of file pkg:/components/UILogic/VideoPlayer.brs
=================================================================
=================================================================
Warning occurred while setting a field of an RoSGNode
-- Type mismatch occurred when setting the "change" field of a "ContentNode" node
   at line 171 of file pkg:/components/UILogic/VideoPlayer.brs
=================================================================
=================================================================
Warning occurred while setting a field of an RoSGNode
-- Tried to set nonexistent field "jumptoitem" of a "Video" node
   at line 239 of file pkg:/components/UILogic/VideoPlayer.brs
=================================================================
```

**Reply by rrirower** [reply]
Thanks for the info. I believe those messages are benign, but, I'll take a closer look.

**Reply by rrirower** [reply]
Btw... If you haven't already, please add your experience with this project to the success stories discussion.

**Comment by rrirower** [comment]
Intra channel search has been resolved in version 1.0.0.1.
